task1/src/Counter.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+/*
+Counter Using React.Component
+ */
+export class Counter extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            counter: 0
+        }
+    }
```

Comment:
setting state in the constructor is old-fashioned class based approach

it can be in new style syntax as:
```
state = {
  counter: 0
}
```

without super and constructor at all

Reply:
fixed